src/utils/block.ts

```diff
@@ -0,0 +1,3 @@
+export function blockHexToDecimal(hex: string): number {
```

Comment:
I think there are some instances of this code in sync-checker that could be removed and instead use this central util?